tests/utils/test_scripts.py

```diff
@@ -203,6 +209,37 @@ def test_get_data_module_multi_gpu_batch_size_adjustment(cfg):
     assert data_module.test_batch_size == cfg.training.test_batch_size
 
 
+def test_get_data_module_multi_gpu_batch_size_adjustment_unsupervised(
```

Comment:
I'd recommend pulling `num_gpus` out of the `cfg` variable and then dividing by this variable instead of hard-coding the "2"

Reply:
Will do